calculation/sha_calc/sha_calc/directivity/bea20/distributions.py

```diff
@@ -0,0 +1,314 @@
+from typing import List
+
+import numpy as np
+from scipy import stats
+import matplotlib.pyplot as plt
+from sha_calc.directivity.bea20.EventType import EventType
```

Comment:
import order

Reply:
What does the `order` package do? :rofl:

Reply:
Also, Pycharm says matplotlib never uesd.